modules/oe_webtools_media/tests/src/Functional/WcloudIntegrationTest.php

```diff
@@ -0,0 +1,115 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\oe_webtools_media\Functional;
+
+use Drupal\Tests\media\Functional\MediaFunctionalTestBase;
+
+/**
+ * Tests the Webtools media source wcloud integration.
+ *
+ * @group oe_webtools_media
+ */
+class WcloudIntegrationTest extends MediaFunctionalTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected $defaultTheme = 'stark';
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
```

Comment:
This is protected. Also a few of these modules are already part of the parent classes.